modules/admin/con-rhdh-deployment-config.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assembly:
+// assembly-rhdh-operator-deployment-config
+
+[id="con-rhdh-deployment-config_{context}"]
+= {product} deployment configuration
+
+As of 1.3, the {product} operator exposes a new `rhdh.redhat.com/v1alpha2` API Version of its Custom Resource Definition (CRD). This CRD exposes a generic `spec.deployment.patch` field, which gives you full control over the RHDH Deployment resource. This field can be a fragment of the standard `apps.Deployment` Kubernetes object.
+
+For example, the following specification fragment will:
+
+* Add an additional volume named `my-volume` and mount it under `/my/path` in the Developer Hub application container.
+* Replace the default `dynamic-plugins-root` volume with a persistent volume claim (PVC) named `dynamic-plugins-root`. Note the `$patch: replace` directive, otherwise a new volume will be added.
+* Set the CPU request for the Developer Hub application container to 250m.
+* Set the label `my=true` for the Developer Hub Pod.
+* Add a new `my-sidecar` sidecar container into the Developer Hub Pod.
+
+[NOTE]
+====
+To learn more about merging, see [Strategic Merge Patch](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-api-machinery/strategic-merge-patch.md#basic-patch-format).
```

Comment:
![image](https://github.com/user-attachments/assets/ce461a48-c83d-4665-be81-01e171c983b2)

Whoops, I thought it was markdown, but this is Asciidoc. I guess the link format needs to be adjusted.

Reply:
:facepalm: my bad. Thanks for catching it!